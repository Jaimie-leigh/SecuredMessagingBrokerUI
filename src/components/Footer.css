.footer {
  height: 2em;
  background-color: #005eb8;
  position: fixed;
  left: 0;
  bottom: 0;
  right: 0;
  background: rgb(0, 94, 184);
  background: linear-gradient(90deg, #005eb8 35%, #071d49 110%);
}
