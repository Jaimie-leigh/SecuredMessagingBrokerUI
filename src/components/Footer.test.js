import React, { useState } from "react";
import renderer from "react-test-renderer";
import { render, unmountComponentAtNode } from "react-dom";
import Footer from "./Footer";

let container = null;

beforeEach(() => {
  // set a DOM elemant as render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // clean up on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe("Footer tests", () => {
  test("test renders as per snapshot", () => {
    const tree = renderer.create(<Footer></Footer>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
