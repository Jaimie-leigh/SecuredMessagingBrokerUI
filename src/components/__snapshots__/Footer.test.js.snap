// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests test renders as per snapshot 1`] = `
<footer
  className="footer"
/>
`;
